docs/user/user-defined-redirects.rst

```diff
@@ -128,6 +130,17 @@ Similarly, if you maintain several branches of your documentation (e.g. ``3.0``
 ``latest``) and decide to move pages in ``latest`` but not the older branches, you can use
 *Exact Redirects* to do so.
 
+You can use an exact redirect to migrate your documentation to another domain,
```

Comment:
This section needs some subsections. There's a lot of content here, I think a few subsections would make it easier to follow. Probably headers with what each example is doing?

Reply:
I have put the examples in subsections.

![Screenshot 2022-05-26 at 16-53-47 User-defined Redirects — Read the Docs user documentation 8 0 2 documentation](https://user-images.githubusercontent.com/4975310/170586622-0d37bafb-8776-457c-905b-983b5be648d5.png)

There is a lot of nesting, maybe we could address a more structural change as part of https://github.com/readthedocs/readthedocs.org/issues/9258